src/views/Dashboard/Dashboard.js

```diff
@@ -74,6 +74,13 @@ class Dashboard extends Component {
     }
   }
 
+  exportCodeJson() {
```

Comment:
NIT: if this is all commented out why leave it in?

Reply:
You're right! Removing.